pom.xml

```diff
@@ -85,6 +85,12 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>structs</artifactId>
+      <version>1.8-20170614.081136-1</version> <!-- TODO remove SNAPSHOT before release -->
+      <!-- require at least 1.8 as existing implementations that add support for traits will need JENKINS-44864 -->
```

Comment:
Well…this dep could be on 1.7, and the downstream PRs which actually use that idiom could request the newer version.